### PR TITLE
[Serializer] versionadded directive for name_converter option

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1497,14 +1497,16 @@ If this option is enabled, serialization groups can be defined using annotations
...
 
+.. _reference-serializer-name_converter:
+
 name_converter
 ..............
 
+**type**: ``string``
+
 .. versionadded:: 2.8
     The ``name_converter`` setting was introduced in Symfony 2.8.
```
Comment (javiereguiluz) [inline reference/configuration/framework.rst]: `The xxx setting was introduced ...`  is used 3 times in Symfony Docs, but `The xxx option was introduced...` alternative is used tens of times. Let's normalize that in a separate PR.

**Review by javiereguiluz — APPROVED** [review]
👍

**Comment by wouterj** [comment]
No brainer 👍 

status: reviewed

**Review by wouterj — APPROVED** [review]

File: reference/configuration/framework.rst (hunk 1 above)
Comment (xabbuh) [inline reference/configuration/framework.rst]: see #7007

**Pull request merged**